docs_source_files/content/grid4/components_of_a_grid.ko.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Components of a Grid"
+weight: 2
+---
```

Comment:
Are the components of Grid4 is different from legacy grid ? If not we have introduce Grid 4 with in legacy grid document.

iWe will end up having duplicate documents for `components of grid` and `purposes and functionalities` if there are no changes from previous.

@diemol Can you please suggest on this!